### PR TITLE
Config and initial test

### PR DESCRIPTION
**Pull request by MaciejGrymuza**
1. Added 1st test
2. Added 2 page objects
3. Added initial config of Playwright

### CODE REVIEW

File: tests/poms/LoginPage/LoginPage.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+export default class LoginPage {
+    readonly userName = this.page.getByPlaceholder('Username');
+    readonly password= this.page.getByPlaceholder('password');
+    readonly loginBtn= this.page.getByRole('button', {name:'Login'});
+    readonly credentials= this.page.getByText('Username : Admin');
+
+    constructor(private page: Page){}
+
+    async goTo(){
+        this.page.goto(baseURL!);
```
Comment (dz1dz1on) [inline tests/poms/LoginPage/LoginPage.ts]: You need await this line.

File: tests/features/logIn.spec.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+import { userName, userPwd } from '../data';
+
+
+test.describe('Log in test suite', () => {
+    test('Log in to the app', async ({page}) => {
+        const loginPage = new LoginPage(page);
+        const mainDashboard = new MainDashboard(page);
+
+        await loginPage.goTo();
+        await loginPage.pageLoaded();
```
Comment (dz1dz1on) [inline tests/features/logIn.spec.ts]: You do this twice. The same method is hidden in goTo()

File: tests/poms/MainDashboard/MainDashboard.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    readonly pageTitle = this.page.locator('.oxd-topbar-header-breadcrumb');
+    readonly sidePanel = this.page.locator('.oxd-sidepanel');
+    readonly hideSidePanelBtn = this.page.locator('[role="none"]');
+
+
+    constructor(private page: Page){};
+
+    async pageLoaded(){
+        await this.hideSidePanelBtn.waitFor();
+        await expect(this.pageTitle).toHaveText('Dashboard');
```
Comment (dz1dz1on) [inline tests/poms/MainDashboard/MainDashboard.ts]: Its hard to say if it is good idea to use expect inside PO but in general it is treated as bad habit.

File: tests/poms/LoginPage/LoginPage.ts (hunk 1 above)
Comment (dz1dz1on) [reply tests/poms/LoginPage/LoginPage.ts]: and the one with waitForLoadState()

File: tests/poms/LoginPage/LoginPage.ts [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    async fillUsername(username: string){
+        await this.userName.fill(username)
+    }
+
+    async fillPassword(pwd: string){
+        await this.password.fill(pwd)
+    }
+        
+    async logIn(){
+        await this.loginBtn.click();
```
Comment (dz1dz1on) [inline tests/poms/LoginPage/LoginPage.ts]: I think would be nice to just put:
```
async logIn(username, password){
  await this.userName.fill(username)
  await this.password.fill(pwd)
  await this.loginBtn.click()
}
```

File: tests/data.ts [hunk 5]
```diff
@@ -0,0 +1,3 @@
+export const userName = 'Admin';
+export const userPwd = 'admin123';
```
Comment (dz1dz1on) [inline tests/data.ts]: If those are global const you should use snake case like const USER_NAME = '...'

**Review by dz1dz1on — CHANGES_REQUESTED** [review]

File: tests/poms/LoginPage/LoginPage.ts (hunk 1 above)
Comment (MaciejGrymuza) [reply tests/poms/LoginPage/LoginPage.ts]: fixed

File: tests/features/logIn.spec.ts (hunk 2 above)
Comment (MaciejGrymuza) [reply tests/features/logIn.spec.ts]: fixed

File: tests/poms/MainDashboard/MainDashboard.ts (hunk 3 above)
Comment (MaciejGrymuza) [reply tests/poms/MainDashboard/MainDashboard.ts]: fixed

File: tests/poms/LoginPage/LoginPage.ts (hunk 4 above)
Comment (MaciejGrymuza) [reply tests/poms/LoginPage/LoginPage.ts]: fixed

File: tests/data.ts (hunk 5 above)
Comment (MaciejGrymuza) [reply tests/data.ts]: fixed

**Pull request merged**